advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -595,6 +603,165 @@
    "source": [
     "predictor.delete_endpoint()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explainability with Amazon SageMaker Clarify\n",
+    "\n",
+    "There are expanding business needs and legislative regulations that require explainations of why a model mades the decision it did. SHAP (SHapley Additive exPlanations) is an approach to explain the output of machine learning models. SHAP values represent a feature's contribution to a change in the model output. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Set parameters for SHAP calculation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seed = 0\n",
+    "num_rows = 500\n",
+    "\n",
+    "#Write a csv file used by SageMaker Clarify\n",
+    "test_explainavility_file = 'test_explainavility.csv'\n",
+    "train.head(num_rows).to_csv(test_explainavility_file, index=False, header=False)\n",
+    "test_explainavility_s3_path = session.upload_data(test_explainavility_file, key_prefix='{}/data'.format(prefix))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Specify computing resources"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker import clarify\n",
+    "\n",
+    "model_name = estimator.latest_training_job.job_name\n",
+    "container_def = model.prepare_container_def()\n",
+    "session.create_model(model_name,\n",
+    "                     role,\n",
+    "                     container_def)\n",
+    "\n",
+    "clarify_processor = clarify.SageMakerClarifyProcessor(role=role,\n",
+    "                                                      instance_count=1,\n",
+    "                                                      instance_type='ml.c4.xlarge',\n",
+    "                                                      sagemaker_session=session)\n",
+    "model_config = clarify.ModelConfig(model_name=model_name,\n",
+    "                                   instance_type='ml.c5.xlarge',\n",
+    "                                   instance_count=1,\n",
+    "                                   accept_type='text/csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Run a SageMaker Clarify job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shap_config = clarify.SHAPConfig(baseline=X_test.sample(15, random_state=seed).values.tolist(),\n",
```

Comment:
maybe for clarity (and since this is an important design choice):  `baseline` should be separately defined as its own variable, with a brief explanation of this choice and other possibilities?

See for example here:  https://github.com/awslabs/autogluon/blob/master/examples/tabular/interpret/SHAP%20with%20AutoGluon-Tabular%20and%20Categorical%20Features.ipynb